### DISCUSSION TITLE
Will there be an official support for Dart programming language?

### DISCUSSION THREAD

**Discussion by manyopensource** [apple/pkl#261] [General]
Personally I would drop Go and include Dart to the list of officially supported languages.

**Comment by StefMa** [comment]
Why dropping go and not just adding dart?
Personal preference? 🤔

**Reply by manyopensource** [reply]
As I stated, yes, it is personal. But the question is different and doesn't state dropping or adding support.

**Reply by RowMax03** [reply]
Some apps actually need backends xD.

**Comment by bioball** [comment]
Dart language support is not on our roadmap. However, feel free to write a language binding for it!

Here's details about how Pkl's language bindings works: https://pkl-lang.org/main/current/bindings-specification/index.html

**Comment by bioball** [comment]
Closing this as a duplicate of #109
